packages/api/src/controllers/stream.ts

```diff
@@ -1015,6 +1015,22 @@ app.post("/:id/heartbeat", authorizer({ anyAdmin: true }), async (req, res) => {
   res.status(204).end();
 });
 
+app.put(
```

Comment:
I like when we're a bit more generic about this stuff — would it be possible to add it as a field to the PATCH handler so we don't need a new endpoint entirely? We can still check if the user is an admin if that field is present in the payload.

Reply:
Yeah that's fair enough, I was considering that way so I'll go for it 👍